admin_manual/configuration_files/file_sharing_configuration.rst

```diff
@@ -15,7 +15,12 @@ Configure your sharing policy on your Admin page in the Sharing section.
 * Check ``Set default expiration date for shares`` to set a default expiration date
   on local user and group shares.
 * Check ``Enforce expiration date`` to always enforce the configured expiration date
-  on local user and group shares. Users will not be able to adjust or remove the expiration date.
+  on local user and group shares. Users will not be able to set the expiration date further
+  in the future than the enforced expiration date, although they will be able to set a more
+  recent date. Also note that they will also be able to update the expiration date again at
```

Comment:
Allow a new paragraph here for better readability

Reply:
That was my original idea :-) But I do not know the syntax, and thus how to add a new paragraph that is shown in the list item. Suggestions welcome :-)

Reply:
I pushed a new revision with the note moved into a _Note_ box.